src/client/opamCommands.ml

```diff
@@ -1180,9 +1180,14 @@ let install =
        definitions, and is typically useful to provide more constrained \
        dependencies that describe a precise development environment."
   in
+  let assume_built =
+    Arg.(value & flag & info ["assume-built"]
+           ~doc:"Just install the package from its pinned directory, but don't \
+                 actually run build.")
+  in
```

Comment:
We should give more explanation and examples of use here: «For use on locally-pinned packages: assume they have already been correctly built, and only run their installation instructions, directly from their source directory. This skips the build instructions and can be useful to install packages that are being worked on. Implies $(i,--inplace-build).»